torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -1121,6 +1159,45 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
               af.pc += inst.X;
             }
           } break;
+          case ENTER: {
```

Comment:
This is pretty repetitive (and a copy/paste) of the CALL node. There are also some issues with ordering. For instance, what happens if an exception occurs in the call to enter? the object is already on the the __enter__/__exit__ stack, but the contract only says that __exit__ will be called _after_ enter.